torch/csrc/jit/codegen/cuda/partition.cpp

```diff
@@ -230,6 +230,11 @@ inline bool isFusibleNode(const Node* node) {
     }
     return false;
   }
+  // linear can't be fused, but for other reasons it needs to be parsible
```

Comment:
ah... Sorry about this. We've separated fusion/profile node registration. So instead of special case on block fusion, we should have updated parser.cpp. Can't believe we haven't run into this earlier :confused: 

Let me push a patch to your PR.